app/grants/models.py

```diff
@@ -572,6 +589,7 @@ def successful_contribution(self, tx_id):
         from marketing.mails import successful_contribution
         self.last_contribution_date = timezone.now()
         self.next_contribution_date = timezone.now() + timedelta(0, round(self.real_period_seconds))
+        self.num_tx_processed = self.num_tx_processed + 1
```

Comment:
any plans to seed the correct data for whats already in the DB?

Reply:
you could add an empty migration that sets `.num_tx_processed` to the number of `contributions.count()`